src/main/java/com/sarality/form/FormData.java

```diff
@@ -65,6 +65,26 @@ public void setInt(FormField field, Integer value) {
     }
   }
 
+  public Double getDouble(FormField field) { return getDouble(field.getViewId()); }
+
+  public Double getDouble(int fieldId) {
+    String value = getValue(fieldId);
+    if (value == null) {
```

Comment:
Can we make this a single check here and everywhere else. Maybe even introduce a method that can be used by all calls.

isNullOrEmpty(String value)